src/benchmarks/perf/cpu/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+# Copyright(c) The Maintainers of Nanvix.
+# Licensed under the MIT License.
+
+[package]
+name = "cpu"
```

Comment:
Please rename it to "matmul". CPU is too generic to convey the meaning of this program :)